### ISSUE TITLE
Having multiple copies of Cluster blocks the ability to preselect Clusters components to be updated

### ISSUE THREAD

**Issue by jakubczajczynski** [nicolaasburgers/definition-library-releases#35]
When there is only one copy of the given Cluster component on GH Canvas I can select it and go to the Library -> Update panel. If I make a copy of that Cluster I can no longer update the copy nor the original Cluster. Thankfully, there is still the way to update/revert - user needs to have none of the GH components selected and then from Library -> Update can open the global version of Update dialog with all things that can be updated.
Of course updating with preselected GH Cluster components is more comfortable and hopefully this can be fixed.

https://github.com/user-attachments/assets/6ec933c3-e346-4ed4-8a3f-f4b4c12d05aa

**Comment by nicolaasburgers** [comment]
Yes that is by design where you either need to have all the copies selected for the "Update selected" option to appear, although I'm reviewing it now in the version I'm working on.  

So from your perspective, am I correct in thinking you'd like to do this?:
- select a single cluster (out of multiple copies)
- then see the "Update selected" option
- if you update it, then ALL the copies (including those that aren't selected) are updated accordingly?

OR - on the last point - should just the selected clusters be updated and if there are other copies, disentangle them and leave them untouched?

**Comment by jakubczajczynski** [comment]
> you either need to have all the copies selected for the "Update selected"

I don't see it on my computer.
![image](https://github.com/user-attachments/assets/ec8eac68-9fc1-4538-9fe6-a65c6a7aea35)

Maybe this is not as big of a problem in the real world scenario and it's more emphasized as I'm testing things.
For example, If I have 5 clusters, all of them in the same version. If I select two of them and proceed with the update/revert. I would like to be asked if:
- I want to update only selected ones (this means disentangle them) and keep the rest 3 as they were.
- Update all 5.

One can argue that having this kind of question is not a good design. It all depends on how many copies of clusters we have in the definition and how often we experiment with different versions.
